src/test/duke/ToDoTest.java

```diff
@@ -0,0 +1,22 @@
+import duke.data.task.ToDo;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ToDoTest {
+    @Test
+    public void testConvertToDoToData() {
```

Comment:
Review all test methods to follow naming format of `featureUnderTest_testScenario_expectedBehavior()` where applicable.